action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
     default: false
   export-env:
     description: Export the secrets as environment variables
-    default: false
+    default: true
```

Comment:
I recall that making `export-env` an opt-in (so default `false`) was a conscious decision, which I don't think we should just revert.